df.history.xml

```diff
@@ -206,14 +206,22 @@
     </struct-type>
 
     <enum-type type-name='histfig_flags'>
-        <enum-item/>
-        <enum-item/>
+        <enum-item name='revealartwork'/>
+        <enum-item name='equipmentcreated/>
```

Comment:
It would probably be better to use underscores (e.g. `reveal_artwork`, `equipment_created`, etc.) here to match other names.